cdhweb/pages/templates/cdhpages/snippets/search_result.html

```diff
@@ -0,0 +1,15 @@
+{% load humanize wagtailcore_tags cdh_tags %}
+{# Single search result on site search page #}
+
+{% spaceless %}
+{% autoescape off %}
+<li class="{{ page.get_url|url_to_icon }}">
```

Comment:
I don't see anything in our currently available icons that would make sense. Hopefully it won't be so long until we're able to revisit and customize styles by type some, so we won't have this forever.  

If you have any ideas of an icon that you think would work, we could suggest it to Gissoo and add to her list of potential new icons. I wondered about a writing icon or pen, but other things could work (and even within our posts we have a lot of variety, too).

Reply:
The one icon that I did think was kind of appealing is the seed — but maybe that's just because we don't use that icon very much and I like it. What do you think, could it make sense? (sometimes blog posts are like seedlings for other work...)

Reply:
I think I'd rather keep the "doc" vs "seed" until we figure out a better alternative...agree that nothing else seems appropriate